recipe/conda_build_config.yaml

```diff
@@ -424,7 +424,9 @@ giflib:
 glew:
   - 2.1
 glib:
-  - 2.58
+  - 2.58  # [not (osx and arm64)]
+  # Can't update to 2.66 on `osx and x86_64` yet because that requires a newer macOS version than 10.9
```

Comment:
This was apparently fixed: https://github.com/conda-forge/conda-forge-pinning-feedstock/issues/1026#issuecomment-747840566